serviceWrappers/linux/kLiveController

```diff
@@ -57,7 +57,7 @@ loadNvm() {
 
 start() {
     [ -d ${LOG_DIR} ] || mkdir -p ${LOG_DIR}
-    loadNvm
+    #loadNvm
```

Comment:
Currently prodit does work with rpm. And i cannot force them to that. Since they are pretty bust team and the ROI of that is negative
You can put a env. Val  that can define if to use deault log folder and if to use nvm so you are comptaible with other enviroments and deployment methods